### DISCUSSION TITLE
그래픽 요소 추가

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [Joseph-Cha/JustFight_Client#21] [General]
좁게 보더라도 폰트, 게임 배경, 캐릭터, 몬스터 등 그래픽 요소를 이제 슬슬 고민을 하려고 합니다. 적절한 그래픽 요소를 찾는 것에도 시간이 다소 필요할 것 같아요. 
그리고 추가로 배경음, 타격음, 이펙트 소리 등등도 찾아야 할 것 같습니당.

**Comment by jongfeel** [comment]
네 그래픽 적인 요소는 얼마든지 취향에 맞게 작업해서 추가하시면 좋습니다.

다만 이런 리소스 파일들은 정적인 파일이고 코드 레벨이 아니기 떄문에 pull request를 통한 코드 리뷰가 필요 없긴 합니다.

그래서 main branch에 자유롭게 추가해서 작업하시면 될 거 같아요.
